src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,112 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { request } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addFilm }) => {
+  const [query, setQuery] = useState('');
+  const [film, setFilm] = useState(null);
+  const [error, setError] = useState(false);
+
+  const handleSearch = () => {
+    request(query)
+      .then((result) => {
+        if (result.Response === 'False') {
+          setFilm(null);
+
+          return setError(true);
```

Comment:
```suggestion
         setError(true);
```
No need to return the function. You are just setting the value